com/zoolatech/java/solomiia_tymoshchuk/src/lecture1/tasks/_1/Task1.java

```diff
@@ -0,0 +1,24 @@
+package com.zoolatech.java.solomiia_tymoshchuk.src.lecture1.tasks._1;
```

Comment:
1. please remove folder `com/zoolatech`.
2. please remove `java.solomiia_tymoshchuk.src` from the package name